test/test.rs

```diff
@@ -142,13 +142,11 @@ fn default_behavior() {
     let script = get_hook_script(&root, "pre-push").unwrap();
```

Comment:
I tried format check:

```
rustfmt --check test/test.rs
```

And it does not report anything. I also tried

```
rustfmt test/test.rs
```

But it does nothing (it means it's already formatted).

How did you format this?

Reply:
As some other people have pointed out to me, my choice of nightly rustfmt is not the best option.

Sorry for bothering you, that was a test run to collect important feedback, which have indeed been collected.
I'll make the adjustment before running it in public space again.

Reply:
I understood the situation. I'm using stable `rustfmt`. Formatting codebase with nightly `rustfmt` is not acceptable for this project since this project does not use nightly toolchain.
For now, I'm closing this PR. When you make some adjustment, please make PR again and I'll be happy to review it. Thank you for your work.